contracts/erc20-bridge-sampler/contracts/src/ERC20BridgeSampler.sol

```diff
@@ -435,14 +436,14 @@ contract ERC20BridgeSampler is
     }
 
     /// @dev Sample sell quotes from an arbitrary on-chain liquidity provider.
-    /// @param providerAddress Address of the liquidity provider contract.
+    /// @param registryAddress Address of the liquidity provider registry contract.
     /// @param takerToken Address of the taker token (what to sell).
     /// @param makerToken Address of the maker token (what to buy).
     /// @param takerTokenAmounts Taker token sell amount for each sample.
     /// @return makerTokenAmounts Maker amounts bought at each taker token
     ///         amount.
     function sampleSellsFromLiquidityProvider(
```

Comment:
As discussed, this should be `sample(Sells|Buys)FromLiquidityProviderRegistry()`